### PR TITLE
update status column

### PR DESCRIPTION
**Pull request by thaonp104**
## Related Tickets:
- [26223](https://edu-redmine.sun-asterisk.vn/issues/26223)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by thaonp104** [comment]
ready

File: database/seeds/CourseUserSeeder.php [hunk 1]
```diff
@@ -15,12 +15,12 @@ public function run()
         $faker = Faker\Factory::create();
         $users = User::where('role_id', 1)->pluck('id')->toArray();
         foreach ($users as $u) {
```
Comment (huyenttm-0901) [inline database/seeds/CourseUserSeeder.php]: `$u` là gì em?

File: database/seeds/SubjectUserSeeder.php [hunk 2]
```diff
@@ -20,17 +20,17 @@ public function run()
             foreach ($subjects as $s) {
```
Comment (huyenttm-0901) [inline database/seeds/SubjectUserSeeder.php:22]: tương tự

File: database/seeds/TaskSeeder.php [hunk 3]
```diff
@@ -15,14 +15,14 @@ public function run()
         $faker = Faker\Factory::create();
         $subjectUsers = SubjectUser::all();
         foreach ($subjectUsers as $i) {
```
Comment (huyenttm-0901) [inline database/seeds/TaskSeeder.php]: tương tự

**Comment by thaonp104** [comment]
ready

**Pull request merged**